src/events/components/SearchModule/DateRangeInput.tsx

```diff
@@ -82,7 +82,7 @@ export const DateRangeInput: FC<IProps> = ({
   };
   
   const formatDate = (day) => {
-    return day.getDate() + ". " +  MONTHS[day.getMonth()] + " " + day.getFullYear();
+    return day.getDate() + ". " +  MONTHS[day.getMonth()].toLowerCase() + " " + day.getFullYear();
```

Comment:
Use DateTime#toFormat ?
https://github.com/moment/luxon/blob/master/docs/formatting.md

Reply:
Just updating this comment as `moment` now is removed. How to use the `date-fns` formatter: https://date-fns.org/v1.28.0/docs/format.

```javascript
import { format } from 'date-fns';
import * as locale from 'date-fns/locale/nb';
```

```javascript
return format('D. MMM YYYY', { locale });
```

Reply:
Isn't that redundant because of `DateTime.format()` inclusion in DateTime? Just use `DateTime.fromJSDate(day: Date)` on the incoming date

Reply:
Nvm, thought that we used `date-fns` and not `luxon`. My bad. Removed my comment. (Was also a little confusing that we removed `moment` and used `luxon` instead, meaning we still use moment - just a more [modernized version](https://github.com/moment/luxon/blob/master/docs/why.md) of it.)